README.md

```diff
@@ -1,6 +1,6 @@
 # Awesome-Books
```

Comment:
- [ ]  Your README file currently lacks an MIT license. Please add it as recommended from this [template](https://github.com/microverseinc/readme-template)